dateparser/languages/validation.py

```diff
@@ -63,9 +69,10 @@ def _validate_word_spacing(cls, language_id, info):
 
         value = info['no_word_spacing']
         if value not in [True, False]:
-            cls.logger.error("Invalid 'no_word_spacing' value %(value)r for '%(id)s' language:"
-                             " expected boolean",
-                             {'value': value, 'id': language_id})
+            cls.get_logger().error(
+                "Invalid 'no_word_spacing' value %(value)r for '%(id)s' "
+                "language: expected boolean",
```

Comment:
You mean, why is it not like this? 

    cls.get_logger().error(
        "Invalid 'no_word_spacing' value %(value)r for '%(id)s' language: "
        "expected boolean",

I did it kind of automatically because there were lot of entries. Picked language word to go to next row, for no particular reason.